lib/inject.ex

```diff
@@ -8,19 +8,21 @@ defmodule Inject do
     :ok
   end
 
-  def i(mod) do
-    inject(mod)
+  defmacro inject(mod) do
+    quote bind_quoted: [mod: mod] do
```

Comment:
since mod actually is not a module but a tuple describing an AST, should this variable name be changed?

well actually I think it's going to be a atom when I think about it, so maybe `mod` makes sense?